test/blackbox-tests/test-cases/meta-gen/run.t

```diff
@@ -1,4 +1,9 @@
   $ dune runtest --force --display short
+  File "dune", line 42, characters 13-34:
```

Comment:
Could you add the thing to disable the warning in these tests? It's just adding unrelated noise otherwise

Reply:
Yes, will do.